setup.py

```diff
@@ -1,11 +1,31 @@
 #!/usr/bin/env python
-#
+import os.path
 
 import setuptools
 
+from sprockets.mixins.amqp import __version__
```

Comment:
This will fail if ``pika`` is not installed already since it's imported in ``__init__.py``